cmd/sentry/sentry/sentry_multi_client.go

```diff
@@ -476,6 +480,9 @@ func (cs *MultiClient) newBlock66(ctx context.Context, inreq *proto_sentry.Inbou
 				propagate = true
```

Comment:
Height is only a proxy for Terminal Total Difficulty. Terminal PoW blocks might have different heights. I guess it's not a big deal, but let's leave a comment in the code that the logic not entirely accurate.